src/client/containers/layout.js

```diff
@@ -130,26 +132,53 @@ class Layout extends React.Component {
 			</span>
 		);
 
+		const showPrivilegeDropdown = user.privs > 1;
+		const adminOptions = (
+			<>
+				<NavDropdown.Item href="/admin-panel">
+					<FontAwesomeIcon className="margin-right-0-3" icon={faUserGear}/>
+					Admin Panel
+				</NavDropdown.Item>
+				<NavDropdown.Item href="/admin-logs">
+					<FontAwesomeIcon className="margin-right-0-3" icon={faNewspaper}/>
+					Admin Logs
+				</NavDropdown.Item>
+			</>
+		);
+
+		const relationshipTypeEditorOptions = (
+			<>
+				<NavDropdown.Item href="/relationship-type/create">
+					Relationship Type Editor
```

Comment:
I like the idea of combining these two icons. 
However, this is how it would look when we hover over the option:
![rel-edit-icon-selected](https://github.com/metabrainz/bookbrainz-site/assets/56965261/04b4c9a3-f943-4b90-9066-54bac689157a)

Another thing is that I think a black and white icon would be more consistent, but this type of layering will probably not be enough to create a black and white icon, as it would not look good in areas where they overlap:
![Rel_edit-icon-layered](https://github.com/metabrainz/bookbrainz-site/assets/56965261/54fb39de-fc3a-433f-9450-f4f291b10741)

We might have to create our own version using these two icons.

Reply:
Ideally we would want the link icon to be white in color in places where it overlaps with the pencil icon. Then I think it would look nice. What do you think?